AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -10,139 +10,103 @@ import AirCastingStyling
 import Resolver
 
 struct ChooseSessionTypeView: View {
-    @State private var isInfoPresented: Bool = false
-    @State private var isTurnBluetoothOnLinkActive = false
-    @State private var isTurnLocationOnLinkActive = false
-    @State private var isPowerABLinkActive = false
-    @State private var isMobileLinkActive = false
-    @State private var didTapFixedSession = false
-    @State private var startSync = false
-    @State private var alert: AlertInfo?
-    var viewModel: ChooseSessionTypeViewModel
     @EnvironmentObject private var tabSelection: TabBarSelection
-    @EnvironmentObject var selectedSection: SelectSection
     @EnvironmentObject private var emptyDashboardButtonTapped: EmptyDashboardButtonTapped
     @EnvironmentObject private var finishAndSyncButtonTapped: FinishAndSyncButtonTapped
-    @Injected private var networkChecker: NetworkChecker
-    @InjectedObject private var bluetoothManger: BluetoothManager //TODO: Fix this (see usage) - move to VM
+    @StateObject var viewModel: ChooseSessionTypeViewModel
+    @InjectedObject private var featureFlagsViewModel: FeatureFlagsViewModel
     
     var shouldGoToChooseSessionScreen: Bool {
         (tabSelection.selection == .createSession && emptyDashboardButtonTapped.mobileWasTapped) ? true : false
     }
     var shouldGoToSyncScreen: Bool {
         (tabSelection.selection == .createSession && finishAndSyncButtonTapped.finishAndSyncButtonWasTapped) ? true : false
     }
-    @InjectedObject private var featureFlagsViewModel: FeatureFlagsViewModel
-
+    
+    init(sessionContext: CreateSessionContext) {
+        self._viewModel = .init(wrappedValue: .init(sessionContext: sessionContext))
+    }
+    
     var body: some View {
-        if #available(iOS 15, *) {
```

Comment:
Did you replace it with anything? I can't find it and the view doesn't work on ios14.